README.md

```diff
@@ -1,2 +1,40 @@
 # MoMEMta-MaGMEE
 MoMEMta - MadGraph Matrix Element Exporter
+
+## Install
+
+For the moment, you need a specific version of MadGraph5_aMC@NLO (MG5) for the exporter to work. You can retrieve it [here](https://code.launchpad.net/~maddevelopers/mg5amcnlo/plugin_mode).
+
+Once your copy of MG5 is setup, go the the `PLUGIN` subfolder. There, download the MoMEMta-MaGMEE plugin (either by cloning the git repository, or by downloading and extracting the archive). 
+
+You are now ready to export matrix elements to use in MoMEMta!
+
+## Usage
+
+Example:
+
+- Go to the MG5 folder
+- Launch MG5: `bin/mg5_aMC`
+- Generate your favourite process: `generate p p > t t~, (t > w+ b~, w+ > e+ ve), (t~ > w- b~, w- > e- ve~)`
+- Output the process: `output MoMEMta myHappyME`. `myHappyME` will be used to define the namespace in which the matrix element code is enclosed, so it has to be a valid C++ variable name.
+
+You now have a folder called `myHappyME` containing the model's code, a parameter card and the matrix element's code itself. 
+
+To build the code and use it in your project, you need to have MoMEMta installed on your system (see the MoMEMta documentation for more details). Then, do:
+```
+cd myHappyME
+mkdir build
+cd build
+cmake ..
+make -j 4
+```
+This generates a shared library that can be dynamically loaded by MoMEMta (using the `load_modules()` function in the Lua config file).
+
+The following options are available when configuring the the build (when running `cmake ..`):
+- `-DCMAKE_PREFIX_PATH=(path)`: Path to your installation of MoMEMta. Use this if your version of MoMEMta was installed locally and not in your system directories
```

Comment:
I'd prefer to document the `MOMEMTA_INCLUDE_DIR` variable instead of tweaking `CMAKE_PREFIX_PATH`.